### ISSUE TITLE
Add php sessions for logging in/using the admin area

### ISSUE THREAD

**Issue by owenvoke** [PXgamer/kastatus#1]
Need to add PHP session usage for the login/admin areas.

So, when logging in it sets the session with the following details:

``` php
$_SESSION['user'] = (object) [
    "id" => "%USERID%",
    "username" => "%USER%",
    "loginDate" => "%DATE%"
];
```

And on the admin page, check that `$_SESSION['user']` is set and isn't equal to `null`.

This is quite an easy one, so anyone who wants it, just comment below. :smile:

If you need any help, let me know.

**Comment by dhwanilvyas** [comment]
I would like to work on it. Can I know the tables and their schema in the database?

**Comment by owenvoke** [comment]
Hi @dhwanilvyas, the database dump can be downloaded from the releases page:
https://github.com/PXgamer/kastatus/releases/latest

**Comment by owenvoke** [comment]
Fixed in #3
